packages/components/src/molecules/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useRef } from 'react'
+import type { FocusEvent, HTMLAttributes } from 'react'
+
+import { Icon } from '../..'
+import type { ButtonProps } from '../..'
+
+export type LinkButtonProps = HTMLAttributes<HTMLAnchorElement> & ButtonProps
+
+function LinkButton({
+  icon,
+  inverse,
+  children,
+  disabled,
+  iconPosition,
+  variant = 'primary',
+  testId = 'fs-link-button',
+  ...otherProps
+}: LinkButtonProps) {
+  const linkRef = useRef<HTMLAnchorElement | null>(null)
+
+  function onFocus(e: FocusEvent) {
+    e.preventDefault()
+
+    if (disabled) {
+      linkRef.current?.blur()
+    }
+  }
+
+  return (
+    <a
+      ref={linkRef}
+      onFocus={onFocus}
+      data-fs-button
+      data-fs-link-button
+      data-testid={testId}
+      data-fs-button-variant={variant}
+      data-fs-button-inverse={inverse}
+      data-fs-button-disabled={disabled}
+      {...otherProps}
+    >
+      {iconPosition === 'left' && <Icon component={icon} />}
+      {children}
+      {iconPosition === 'right' && <Icon component={icon} />}
+    </a>
```

Comment:
Shouldn't we use the Link component in this component? In the staters we can't use the frameworks Link in this way.

Reply:
As discussed before, we'll stick with the anchor tag for now while we don't resolve the style precedence issue. In the future, we will come back to this component.